tools/cdb2sql/cdb2sql.cpp

```diff
@@ -299,13 +299,22 @@ static char *db_generator(const int state, const char *sql)
         if (isadmin)
             flags |= CDB2_ADMIN;
 
+        struct timeval start, end;
+
+        gettimeofday(&start, NULL);
         rc = cdb2_open(&cdb2h_2, dbname, type, flags);
+        gettimeofday(&end, NULL);
+        fprintf(stderr, "Connect %d\n",
```

Comment:
Are these needed, or should they be printed only if (debug_trace)?

Reply:
I will remove them, it was just instrumentation during performance review

Reply:
removed